cdap-ui/app/features/tracker/directives/lineage/lineage.js

```diff
@@ -58,18 +58,22 @@ function LineageController ($scope, jsPlumb, $timeout, $state, LineageStore, myT
         lineWidth: 2,
         strokeStyle: 'rgba(0,0,0, 1)'
       },
-      Connector: [ 'Flowchart', {gap: 0, stub: [10, 15], alwaysRespectStubs: true, cornerRadius: 3} ],
-      Endpoints: ['Blank', 'Blank']
+      // Connector: [ 'Flowchart', {gap: 0, stub: [10, 15], alwaysRespectStubs: true, cornerRadius: 3} ],
```

Comment:
Not yet. Might need to revert back to this depending on what we decide with lineage. Lineage will go through couple more iterations